modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeSetResultsSetTotalMethodCheck.java

```diff
@@ -43,6 +43,31 @@ protected String[] getValidExtensions() {
 		return new String[] {"java", "jsp", "jspf"};
 	}
 
+	private String _addTryCatch(
+		String newContent, String lastMethodCall, String stringBuilder) {
+
+		String newClass = StringUtil.replace(
+			newContent, lastMethodCall, stringBuilder);
```

Comment:
newContent = StringUtil.replace(
			newContent, lastMethodCall, stringBuilder);

remove this variable